Resources/Prototypes/_NF/Research/industrial.yml

```diff
@@ -1,5 +1,18 @@
 # Tier 1
 
+- type: technology
+  id: ConstructionBags
+  name: research-technology-construction-bags
+  icon:
+    sprite: _NF/Objects/Specific/Engineering/construction_bag.rsi
+    state: icon
+  discipline: Industrial
+  tier: 1
+  cost: 5000
+  recipeUnlocks:
+  - ConstructionBag
```

Comment:
Technically, the ConstructionBag recipe isn't used in the protolathe, and isn't unlocked by this in the engineering techfab since it's a static recipe.  I think it could be removed from this list.

Reply:
Sure, can do